src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,92 @@
+package duke;
+
+import duke.command.Command;
+import duke.command.DeadlineCommand;
+import duke.command.DeleteCommand;
+import duke.command.DoneCommand;
+import duke.command.EventCommand;
+import duke.command.ExitCommand;
+import duke.command.FindCommand;
+import duke.command.ListCommand;
+import duke.command.ShowCommand;
+import duke.command.TodoCommand;
+import duke.exception.DukeException;
+import duke.exception.WrongInputException;
+
+public class Parser {
+
+    /**
+     * Returns the command based on user's input.
+     * @param command User's input.
+     * @return Corresponding command.
+     * @throws DukeException If user's input is not compatible to any command.
+     */
+
+    public static Command parse(String command) throws DukeException {
```

Comment:
The parse method's structure is very clear!

Reply:
Thank you for your kind words!